Makefile

```diff
@@ -1,4 +1,5 @@
-SHELL := /bin/bash
+SHELL := /bin/bash -o pipefail
+.SHELLFLAGS = -ec
```

Comment:
Might as well include the comment from here: https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.7.1/#gov3-set-the-makefiles-shell-variable-to-the-systems-bash-binary

It will help us to remember what this is for.